Makefile

```
@@ -72,12 +72,25 @@ sphinx-help:
 # The .PHONY directive tells make that `apidocs` and `html` are labels for 
 # commands. `apidocs: html` allows us to generate docs by running 
 # `make apidocs` instead. 
-.PHONY: apidocs html 
-
+.PHONY: docs-comprehensive apidocs html 
+
+# Start an interactive server to test docs locally
+test-docs-comprehensive: docs-comprehensive 
```

```suggestion
develop-docs-comprehensive: docs-comprehensive 
```
The term "test" implies it's good as part of an automated test suite, which `npm start` is not.  So I'd call it develop.